_data/baic/schedule/fall22.yml

```diff
@@ -0,0 +1,50 @@
+- abstract: "Biweekly Antiracism and Inclusion Conversations Joint Kick-off (Staff, Faculty and Grad Students)"
+  date: 2022-09-09
+  summary: Coming soon
```

Comment:
Is this supposed to have something here :)

Reply:
It was coming soon when I wrote it up so it was meant to be just that. 😄 
I got the new content from Rajashree's mail. 
Updating.

Reply:
Updated it. 
Added time.